src/nxdoc/nuxeo-add-ons/nuxeo-retention-management/nuxeo-retention-installation/nuxeo-retention-installation-standard.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Standard mode
+description: Learn how to install and configure the Nuxeo Retention addon in the standard mode.
+review:
+    comment: ''
+    date: '2019-08-05'
+    status: 'ok'
+labels:
+    - lts2019-wip
+    - grenard
+    - jaubenque
+    - retention-management
+toc: true
+tree_item_index: 1000
+---
+
+## Before You Start
+
+This page gives all the necessary steps to install the Retention Management addon with the [Standard mode]({{page page='nuxeo-retention-management'}}#configuration-modes).
+
+## Prerequisites
+
+You can use all the [file storages]({{page page='file-storage'}}) supported by Nuxeo Platform.
+
+## Installation
+
+{{{multiexcerpt 'MP-installation-easy' space='nxdoc' page='Generic Multi-Excerpts'}}}
+
+## Configuration
+
+### {{> anchor 'standard-amazon-s3'}} Amazon S3
+
+In Standard mode, there are 3 supported configurations with Amazon S3:
+
+- Records are stored in the same S3 bucket than the standard documents.</br>
+  [<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More info](#s3-one-bucket-configuration)
+- Records are stored in a dedicated S3 bucket.</br>
+  [<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More info](#s3-2-buckets-configuration)
+- Records are stored in a dedicated S3 bucket, with [**Amazon S3 Object Lock**](https://docs.aws.amazon.com/AmazonS3/latest/dev/object-lock.html) in Governance mode.</br>
```

Comment:
"Records are stored in a dedicated S3 bucket, with [**Amazon S3 Object Lock**]" is enough